pyrefly/lib/lsp/wasm/hover.rs

```diff
@@ -176,16 +179,34 @@ impl HoverValue {
             .clone()
             .unwrap_or_else(|| self.type_.as_hover_string());
 
+        let is_function = self.type_.is_function_type();
+        if is_function && kind_formatted.trim().is_empty() {
+            kind_formatted = "(function) ".to_owned();
+        }
+        let trimmed_type_display = type_display.trim_start();
+        let needs_def_wrapper = trimmed_type_display.starts_with('(');
+        let (heading, body) = if is_function {
+            let function_name = self.name.clone().unwrap_or_else(|| "function".to_owned());
+            let heading = format!("{}{}\n", kind_formatted, function_name);
+            let body = if needs_def_wrapper {
```

Comment:
the other logic for adding `def ` is in pyrefly/crates/pyrefly_types/src/display.rs. can we move this logic there instead?